Mobile and all

msecret opened the pull request:
Combination of billboard, override, ups, footer, adds
mobile styles to ensure things look ok on mobile to finish up
the whole landing homepage.

Probably should merge this and forget the rest of the PRs.

Commit:
Sets an override on the usa disclaimer

Requires being overridden because uswds sets a different root
font-size on <html> which effects all rem units.

Commit:
Billboard and section components

The billboard is a large display section of the site with
mainly text.

The section is the different colored background sections, will
eventually have the diamond shapes as in mockup.

Commit:
Few more changes to section

To allow for different section elements to stack on top
of eachother with the right spacing.

Commit:
Three up and four ups

Not the best name for this component, but it's a component that
creates either 3 or 4 content items side by side and will likely
have an icon for each one.

Could potentially put the 3 and 4 together and use modifier
classes.

Commit:
Merge remote-tracking branch 'origin/master' into ups

Commit:
Contact form styling

Commit:
Footer.

Commit:
Mobile fixes for landing and rest

These changes build on billboard, override, ups, footer, adds
mobile styles to ensure things look ok on mobile to finish up
the whole landing homepage.

Commit:
Merge remote-tracking branch 'origin/master' into mobile_and_all

Commit:
Finsih off mobile styles for sidebar

Fixes sidebar to be full height.

jcscottiii commented on src/css/base/typography.scss:
```diff
@@ -7,6 +7,10 @@ h1, h2, h3, h4, h5, h6 {
     margin-top: 0.5rem;
   }
+
+  & + .push {
```
I see only the landing page is using this for contact. 

hmm. usually when i see push / pull classes I see it as `push-X` x being a number of some sorts. just want to know how much to expect the push in the name.

msecret replied:
Yea I want a different name to it because it's different then the standard push. It's a class that signifies that a div should be treated as a block of text.

jcscottiii commented:
maybe `four_up` -> `four-cols` or `four-columns` and 
 `four_up-part` -> `four_cols-col`?
and similar scheme for three_up too.

jcscottiii commented on src/css/base/form.scss:
```diff
@@ -0,0 +1,23 @@
+
+input[type="email"] {
+  font-size: 1rem;
+  padding: .7rem 0.7em;
```
use rem for the second one? and specify all four dimensions or explicitly only do `padding-top` and `padding-right`

jcscottiii commented:
@msecret just a few suggestions above. otherwise looks good!

Commit:
Rename vars and fix values

Renamed push to text-block because push has different connotations in FE
dev and don't fit with this use case.

All units should be rem.
